index.html

```diff
@@ -62,37 +62,47 @@ <h3 class="meal-name">
                     </div>
                 {{~}}
             </div>
-            
         </script>
     </head>
-    
+
     <body>
         <div class="top-container">
-            <h1>TTY:n ruokalistat</h1>
-            
+            <div>
+                <div id="tunilogo"></div>
+
+                <h1>Ruokalistat</h1>
+            </div>
+
             <button id="prev-day" class="nav-button">Edellinen päivä</button>
             <button id="next-day" class="nav-button">Seuraava päivä</button>
         </div>
-        
-        <div id="main-content">
-            
-        </div>
-                
+
+        <div id="main-content"></div>
+
         <!-- build:js js/build.js -->
         <script src="lib/dot.min.js"></script>
         <script src="lib/moment.min.js" charset="utf-8"></script>
         <script src="lib/moment.fi.js" charset="utf-8"></script>
         <script src="js/main.js" charset="utf-8"></script>
         <!-- endbuild -->
-        
+
         <script>
-            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+            ;(function(i, s, o, g, r, a, m) {
+                i["GoogleAnalyticsObject"] = r
+                ;(i[r] =
+                    i[r] ||
+                    function() {
+                        ;(i[r].q = i[r].q || []).push(arguments)
+                    }),
+                    (i[r].l = 1 * new Date())
+                ;(a = s.createElement(o)), (m = s.getElementsByTagName(o)[0])
+                a.async = 1
+                a.src = g
+                m.parentNode.insertBefore(a, m)
+            })(window, document, "script", "//www.google-analytics.com/analytics.js", "ga")
 
-            ga('create', 'UA-69443636-1', 'auto');
-            ga('send', 'pageview');
+            ga("create", "UA-69443636-1", "auto")
```

Comment:
Please use single quotes in JS as per project's conventions

Reply:
I refer to the comment of the project not having proper linting file. This was also done by the auto formatter :(